sn_cli/src/subcommands/register.rs

```diff
@@ -101,11 +101,22 @@ async fn get_registers(names: Vec<String>, client: &Client) -> Result<()> {
         println!("Trying to retrieve Register from {xorname:?}, {tag}");
 
         match client.get_register(xorname, tag).await {
-            Ok(register) => println!(
-                "Successfully retrieved Register '{name}' from {}, {}!",
-                register.name(),
-                register.tag()
-            ),
+            Ok(register) => {
+                println!(
+                    "Successfully retrieved Register '{name}' from {}, {}!",
+                    register.name(),
```

Comment:
using `register.address()` will give both.
actually, it shall be `address` to be used across to be consistent and avoid misleading